openid-sharedsignals-framework-1_0.md

```diff
@@ -1838,12 +1897,12 @@ Event Receiver as a result of the above request:
   "iss": "https://transmitter.example.com",
   "aud": "receiver.example.com",
   "iat": 1493856000,
+  "sub_id": {
+    "format": "opaque",
+    "id": "f67e39a0a4d34d56b3aa1bc4cff0069f"
+  },
   "events": {
     "https://schemas.openid.net/secevent/ssf/event-type/verification":{
-      "subject": {
```

Comment:
I think we said that "subject" in the event will be retained but called out as deprecated.

Reply:
we have now removed this.

Reply:
This is a breaking change, but since the spec is under development, and this part hasn't been used so far, I thought it was safe to remove the field. We can talk about it on the next call if required.